tsdb/compact_test.go

```diff
@@ -737,6 +737,29 @@ func TestCompaction_populateBlock(t *testing.T) {
 				},
 			},
 		},
+		{
+			title: "Break chunks that are greater than 120 in size.",
+			inputSeriesSamples: [][]seriesSamples{
+				{
```

Comment:
You can increase readability by removing some of newlines

Reply:
I adhered to the styling that was in all the previous test cases. Does this change require modifying the style of all the previous test cases?

Reply:
Ideally yes. See https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html

However it is too complex, affect too many things feel free to ignore my comment, it's a minor nit.